tools/pythonpkg/duckdb/experimental/spark/sql/dataframe.py

```diff
@@ -27,6 +28,9 @@ def __init__(self, relation: duckdb.DuckDBPyRelation, session: "SparkSession"):
 
     def show(self, **kwargs) -> None:
         self.relation.show()
+    
+    def toPandas(self) -> PandasDataFrame:
```

Comment:
I think we don't want to have a dependency on `pandas`, what we do in other places is set `"PandasDataFrame"` as the return type annotation and then inside the method do the actual import - so it only fails to import when `toPandas` is used if it's not installed.

Reply:
Good point. Fixed here: `b849a19fc299b6e87a0069e1aacb82630a6671a3`

Reply:
Thanks, though I think the type annotation needs to be in quotes still?
Otherwise this is referencing a type that doesn't exist, can't imagine the type checker to be happy with that?

Reply:
Not fixed properly yet^

Reply:
Ok,  I have tried to fix it in ` 2c5e7b690e61c22c4eec18db15706111fce858f7`. 
I have tried to make it mirror how it works in the session.py with the type checking.